gstd/gstd_pipeline.c

```diff
@@ -375,6 +398,13 @@ gstd_pipeline_set_property (GObject * object,
       }
       self->state = g_value_get_object (value);
       break;
+    case PROP_GRAPH:
+      if (self->graph)
```

Comment:
Try to always use brackets, even if its a single line if. This is a potential source of bugs, eventually someone could add a second line to the conditional without realizing it lacks brackets.